Recipes/Recipes.API/Models/Entities/RecipeEntity.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.ComponentModel.DataAnnotations.Schema;
+using System.ComponentModel.DataAnnotations;
+
+namespace Recipes.API.Models.Entities;
+
+public class RecipeEntity
+{
+    [Key]
+    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
+    public int Id { get; set; }
```

Comment:
Probably should change `Id`  from `int` to `Guid`

Reply:
int generated by a database sequence is ok for me